src/app/NewCollectionEvent.js

```diff
@@ -165,6 +165,8 @@ define([
             //        description
             console.log('app/NewCollectionEvent:onSuccessfulSubmit', arguments);
 
+            this.locationTb.verifyMap.map.setView([40.6389, -111.7034], 10);
```

Comment:
This might be annoying if you are working in a different part of the state. Maybe it's required to get the map back to life?

Reply:
No. Its how the map is when you access it fresh other wise it stays where you were. I'm not sure what is expected.

Reply:
I would expect it to stay where you were. But I don't feel strongly either way on this one.